src/main/java/uk/gov/hmcts/reform/authorisation/validators/ServiceAuthTokenValidator.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.authorisation.validators;
+
+import feign.FeignException;
+import uk.gov.hmcts.reform.authorisation.ServiceAuthorisationApi;
+import uk.gov.hmcts.reform.authorisation.exceptions.InvalidTokenException;
+
+public class ServiceAuthTokenValidator implements AuthTokenValidator {
+
+    private final ServiceAuthorisationApi api;
+
+    public ServiceAuthTokenValidator(ServiceAuthorisationApi api) {
+        this.api = api;
+    }
+
+    @Override
+    public void validate(final String token) {
+        validate(token, new String[0]);
+    }
+
+    @Override
+    public void validate(String token, final String[] roles) {
+        try {
+            if (!token.startsWith(BEARER)) {
+                token = BEARER + token;
```

Comment:
shouldn't it always have `Bearer <space>` at this point?

Reply:
if no1 tampers with it, i guess so. i was testing endpoint locally with some random header values